nodeJS/APIs/APIs.md

```diff
@@ -27,26 +27,26 @@ _However_, REST (an acronym for Representational State Transfer) is a popular an
 
-As mentioned above, you access and manipulate things using the HTTP verbs that you already know about, GET, POST, PUT and DELETE. The usage here is basically what you would expect, but to be clear, lets work through a few examples.
+As mentioned above, you access and manipulate things using the HTTP verbs that you already know about, GET, POST, PUT and DELETE. The usage here is basically what you would expect, but to be clear, let's work through a few examples.
 
 - To get a list of blog posts you would `GET` `/posts`, so in your express app you would respond to  `app.get("/posts")` with a list of all posts.
 - To add a new blog-post you would `POST` to `/posts`.
 - To update the contents of a blog-post you would `PUT` to the URI of the specific post you're updating. `/posts/:postid`
 - To delete a post you would `DELETE` `/posts/:postid`
 
 ### CORS
-The [Same Origin Policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) is an important security measure that basically says "Only requests from the same origin (the same IP address or URL) should be allowed to access this API". (Look at the link above for a couple of examples of what counts as the 'same origin'.) This is a big problem for us, because we are specifically trying to set up our API so that we can access it from different origins, so to enable that we need to set up Cross-origin resource sharing, or CORS.
+The [Same Origin Policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) is an important security measure that basically says "Only requests from the same origin (the same IP address or URL) should be allowed to access this API". (Look at the link above for a couple of examples of what counts as the 'same origin'.) This is a big problem for us because we are specifically trying to set up our API so that we can access it from different origins, so to enable that we need to set up Cross-origin resource sharing, or CORS.
 
 Setting up CORS in express is very easy, there's a middleware that does the work for us. [This article](https://medium.com/@alexishevia/using-cors-in-express-cac7e29b005b) explains everything you need to do, and the official docs can be found [here](https://expressjs.com/en/resources/middleware/cors.html).
 
-For now it is acceptable to just allow access from any origin.. this makes development quite a bit easier but for any _real_ project, once you deploy to a production environment you will probably want to specifically block access from any origin _except_ your front-end website. The documentation above explains how to do this.
+For now, it is acceptable to just allow access from any origin.. this makes development quite a bit easier but for any _real_ project, once you deploy to a production environment you will probably want to specifically block access from any origin _except_ your front-end website. The documentation above explains how to do this.
```

Comment:
"For now, it is acceptable to just allow access from any origin.." <- let's get rid of the extra period here.

Reply:
done

Reply:
ops misread it. let me remove the period